lib/commands/instrument.js

```diff
@@ -48,6 +49,16 @@ exports.builder = function (yargs) {
       type: 'boolean',
       description: 'should nyc exit when an instrumentation failure occurs?'
     })
+    .option('include', {
+      alias: 'n',
+      default: [],
+      describe: 'a list of specific files and directories that should be instrumented, glob patterns are supported'
+    })
+    .option('exclude', {
+      alias: 'x',
+      default: testExclude.defaultExclude,
```

Comment:
Does `nyc instrument input output` honor an `exclude` setting in package.json or .nycrc?

Reply:
Yes, with this change include/exclude settings in `.nycrc` or the `package.json` `nyc` stanza will be honored by `nyc instrument`.  The settings are recovered from the config file/nyc stanza and passed into the `nyc` constructor when creating an instance to run the instrumentation.  From here it will be applied by the filters in 'test-exclude'.

Reply:
So would this item need specific tests?

Reply:
I'd be OK with doing it in a follow-up though as long as you've verified manually that it works.

Reply:
I verified it before posting my response.  To be honest I thought that allowing `include` and `exclude` to be configured via cli, config file or config stanza was the point of this work.